docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -245,6 +245,24 @@ Property Name                                        Description
 ``iceberg.nessie-catalog.default-warehouse-dir``     Default warehouse directory for schemas created without an
                                                      explicit ``location`` property.
                                                      Example: ``/tmp``
+
+``iceberg.nessie-catalog.auth.type``                 The authentication type to use.
+                                                     Available values are ``NONE`` or ``BEARER``.
```

Comment:
Note: 
I didn't support `BASIC` as it is deprecated. 

https://projectnessie.org/tools/client_config/#authentication-settings